src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1906,6 +1906,14 @@ Every bucket is identified by a unique bucket name. The bucket naming should mat
 
 * A bucket name must start with a lowercase letter and cannot terminate with a dash.
 
+=== Object naming requirements
+
+Each Catalog object is identified by a unique object name. The object naming should match the following rules:
+
+* The object name must not be empty, should start and finish with non-whitespace character.
+
+* The object name can contain all characters except forward slash and comma.
+
```

Comment:
Each Catalog object is identified by a unique object name. The object naming has a number of constraints that must be considered:
1. Empty values are not allowed,
2. Must not be any trailing whitespace at the beginning or/and at the end,
3. Accept all characters except forward slash and comma ('/', ',').